config/routes.rb

```diff
@@ -2,5 +2,9 @@
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root 'users#index'
+  get '/users/:id', to: 'users#show'
+
+  get '/users/:user_id/posts', to: 'posts#index'
+  get '/users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
- Optional
You can nest the user and post routes together, since posts will always be associated with User.
Like this. ⬇️

```
resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
end
```

Reply:
Thanks, @muneebulrehman 😊